PWG/EMCAL/READMEemcCorrections.txt

```diff
@@ -498,9 +499,30 @@ virtual Bool_t Run();
 virtual Bool_t UserNotify();
 ~~~
 
-To configure your task in the ``Initialize()`` function with the YAML configuration, you must define the object and then get it via the ``GetProperty("propertyName", property)`` function defined in ``AliEmcalCorrectionComponent``. For an example, see any ``Initialize()`` in any of the correction components.
+To configure your task in the ``Initialize()`` function with the YAML configuration, you must define the
+object and then get it via the ``GetProperty("propertyName", property)`` function defined in
+``AliEmcalCorrectionComponent``. For an example, see ``Initialize()`` in any of the correction components.
 
-Lastly, the task needs to be added to the default YAML configuration file. To do so, add the name of the component (removing "AliEmcalCorrection") to the file, and then set default values for each parameter. Be certain to document what each parameter means! In addition to the component configuration, be certain to add it into the executionOrder node! Otherwise, your task will never be executed!
+For your component to be accessible to the correction task, it must be registered within the framework.
+To do so, add the following to your component header:
+
+```{.cxx}
+// Allows the registration of the class so that it is availble to be used by the correction task.
```

Comment:
Doxygen eats the `//` on this line for no apparent reason, even though the line two below is fine.. Running doxygen locally produces no errors. @jdmulligan , do you have any ideas?

Reply:
See the bottom of this page: http://alidoc.cern.ch/AliPhysics/master/READMEemcCorrections.html